integration/mtcli/list_bundles_test.go

```diff
@@ -0,0 +1,106 @@
+package mtcli_test
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/mt-sre/addon-metadata-operator/pkg/utils"
+	"github.com/stretchr/testify/suite"
+)
+
+type e2eTestSuite struct {
+	suite.Suite
+	mtcliPath string
+}
+
+func TestE2ETestSuite(t *testing.T) {
+	suite.Run(t, &e2eTestSuite{})
+}
+
+func (s *e2eTestSuite) TearDownSuite() {
+	// Remove cache directory
+	err := os.RemoveAll(utils.DefaultCacheDir)
+	if err != nil {
+		fmt.Fprintln(os.Stderr, "Failed to cleanup cache dirs")
+	}
+}
+
+func (s *e2eTestSuite) SetupSuite() {
+	s.mtcliPath = os.Getenv("E2E_MTCLI_PATH")
+	_, err := os.Stat(s.mtcliPath)
+	s.Require().NoError(err, "Cant find mtcli binary at E2E_MTCLI_PATH")
+}
+
+func (s *e2eTestSuite) TestMtcliListBundlesE2E() {
+	type testCaseArg struct {
+		indexImageUrl  string
+		expectedOutput []string
+	}
+
+	testCases := []testCaseArg{
```

Comment:
```suggestion
	testCases := []struct{
	        indexImageUrl  string
		expectedOutput []string
	}{
```
Seems like a cleaner place to use anonymous structs from my eyes. WDYT :)